dockstore-webservice/src/main/java/io/dockstore/webservice/DockstoreWebserviceApplication.java

```diff
@@ -273,6 +275,17 @@ public DataSourceFactory getDataSourceFactory(DockstoreWebserviceConfiguration c
         }
         okHttpClient = builder.cache(cache).connectTimeout(0, TimeUnit.SECONDS).readTimeout(0, TimeUnit.SECONDS)
                 .writeTimeout(0, TimeUnit.SECONDS).build();
+        try {
+            // this can only be called once per JVM, a factory exception is thrown in our tests
+            URL.setURLStreamHandlerFactory(new ObsoleteUrlFactory(okHttpClient));
+        } catch (Error factoryException) {
```

Comment:
Interesting, did this Error only start getting thrown with the changes to this PR? Or are you just being extra cautious?

Reply:
Oh, I didn't add this, it's currently like that in the code

Reply:
Oh, I was only looking at your changes since my last review; I guess you'd removed it then added it back? But yeah, all good, no change.